common/defs/misc/misc.json

```
@@ -87,6 +87,22 @@
             "Daedalus": "https://daedaluswallet.io"
         }
     },
+    {
+        "name": "Cardano Testnet",
+        "shortcut": "tADA",
+        "slip44": 1815,
```

Every testnet should use 1 for SLIP44.

```suggestion
        "slip44": 1,
```